README.md

```diff
@@ -16,7 +15,7 @@ Before installing the extension, make sure you meet the following requirements:
 1. You have a [Codacy account](https://www.codacy.com/signup-codacy).
 2. The repository you’re working on has been [added to Codacy Cloud](https://docs.codacy.com/organizations/managing-repositories/#adding-a-repository).
 3. You have at least [Repository Read permissions](https://docs.codacy.com/organizations/roles-and-permissions-for-organizations/) for the repository you’re working on.
-4. IntelliJ IDEA, version 2023.3.3 or later.
+4. You're using IntelliJ IDEA version 2023.3.3 or later.
```

Comment:
If this matches the [version in gradle.properties](https://github.com/codacy/codacy-intellij-extension/blob/f0ea50985e5c99d3d8a50460ade89ca405e7f3a4/gradle.properties#L15), we can remove this line, as the prerequisite should be checked during installation automatically.